packages/fontaine/test/fixtures/with-comments.scss

```
@@ -0,0 +1,6 @@
+// [Poppins Variable]
+@import "~@fake-fontsource/dm-sans";
+// url(https://example.com/not-a-comment.css)
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)